api/db/seeder/legacySeeder.ts

```diff
@@ -0,0 +1,204 @@
+import { Connection } from 'typeorm';
+import {
+  Instrument,
+  Genre,
+  Musician,
+  Groups,
+  MusicianGroup,
+  Event,
+} from '../../entity';
+
+export async function legacySeeder(connection: Connection): Promise<any> {
+  try {
+    // get All the repo
+    const insRep = connection.getRepository(Instrument);
+    const genRep = connection.getRepository(Genre);
+    const musRep = connection.getRepository(Musician);
+    const groRep = connection.getRepository(Groups);
+    const musGrouRep = connection.getRepository(MusicianGroup);
+    const eveRep = connection.getRepository(Event);
+
+    // Reset all the database for the moment
+    insRep.query('DELETE FROM instrument');
+    genRep.query('DELETE FROM genre');
+    musRep.query('DELETE FROM musician');
+    groRep.query('DELETE FROM groups');
+    musGrouRep.query('DELETE FROM musician_group');
+    eveRep.query('DELETE FROM event');
+
+    console.log('🚮 Reset all the DB tables');
+
+    const metal = genRep.create({ name: 'metal' });
+    const rock = genRep.create({ name: 'rock' });
+    const jazz = genRep.create({ name: 'jazz' });
+
+    await genRep.save([metal, rock, jazz]);
+    console.log('🎵 genres saved');
```

Comment:
Why do you keep this genre entity creation when you have a `genreSeeder `in the `/api/db/seed.ts` file